dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Views/Shared/RestError.cshtml

```diff
@@ -29,10 +29,17 @@
     if (commonError != null)
     {
         ErrorModel deserializeObject = JsonConvert.DeserializeObject<ErrorModel>(commonError.Response.Content);
-        <p>@deserializeObject.Error.Message</p>
-        if (deserializeObject.Error.Details != null)
+        if (deserializeObject != null)
         {
-            <p>@deserializeObject.Error.Details</p>
+            <p>@deserializeObject.Error.Message</p>
+            if (deserializeObject.Error.Details != null)
```

Comment:
Details is of type Object, not string.

---
In reply to: [117381871](https://github.com/Microsoft/healthvault-samples/pull/10#discussion_r117381871) [](ancestors = 117381871)